src/tests/test_dte_data_models.py

```diff
@@ -1253,151 +1376,168 @@ def test_validate_non_empty_bytes_signature_x509_cert_der(self) -> None:
         expected_validation_errors = [
             {
                 'loc': ('signature_x509_cert_der',),
-                'msg': 'Bytes value length is 0.',
+                'msg': 'Value error, Bytes value length is 0.',
                 'type': 'value_error',
             },
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 self.dte_xml_data_1,
                 signature_x509_cert_der=b'',
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
+        validation_errors = assert_raises_cm.exception.errors(
+            include_context=False,
+            include_input=False,
+            include_url=False,
+        )
         self.assertEqual(len(validation_errors), len(expected_validation_errors))
-        for expected_validation_error in expected_validation_errors:
-            self.assertIn(expected_validation_error, validation_errors)
+        self.assertEqual(validation_errors, expected_validation_errors)
 
     def test_validate_no_leading_or_trailing_whitespace_characters_emisor_giro(self) -> None:
         expected_validation_errors = [
             {
                 'loc': ('emisor_giro',),
-                'msg': "('Value has leading or trailing whitespace characters.', ' NASA ')",
+                'msg': (
+                    "Value error, ("
+                    "'Value has leading or trailing whitespace characters.', ' NASA '"
+                    ")"
+                ),
```

Comment:
Reformat to reduce diff size.

```suggestion
                'msg': (
                    "Value error, "
                    "('Value has leading or trailing whitespace characters.', ' NASA ')"
                ),
```